cdc/scheduler/internal/v3/replication/replication_set.go

```diff
@@ -823,15 +829,15 @@ func (r *ReplicationSet) handleMoveTable(
 	// Ignore move table if it has been removed already.
 	if r.hasRemoved() {
 		log.Warn("schedulerv3: move table is ignored",
-			zap.Any("replicationSet", r), zap.Int64("tableID", r.TableID))
+			zap.Any("replicationSet", r), zap.Int64("tableID", r.Span.TableID))
 		return nil, nil
 	}
 	// Ignore move table if
 	// 1) it's not in Replicating state or
 	// 2) the dest capture is the primary.
 	if r.State != ReplicationSetStateReplicating || r.Primary == dest {
 		log.Warn("schedulerv3: move table is ignored",
-			zap.Any("replicationSet", r), zap.Int64("tableID", r.TableID))
+			zap.Any("replicationSet", r), zap.Int64("tableID", r.Span.TableID))
```

Comment:
It's better to put `tableID` before `replicationSet`, since it's shorter, easy to read.